.wordlist.txt

```diff
@@ -2,9 +2,11 @@
 # modified. Add any custom terms to .custom_wordlist.txt instead.
 
 addons
+aws
```

Comment:
Try to avoid adding this here so that we can use the spelling check to maintain consistency of usage (only use 'AWS' instead of 'aws'). 
In this case putting ``aws-cn`` in double back-ticks should solve the error.

Reply:
fixed